cloudera/datadog_checks/cloudera/api/api_v7.py

```diff
@@ -105,9 +106,30 @@ def _collect_cluster_metrics(self, cluster_name, tags):
         query = f'SELECT {metric_names} WHERE clusterName="{cluster_name}" AND category=CLUSTER'
         self._query_time_series('cluster', cluster_name, query, tags)
 
-    def _collect_hosts(self, cluster_name):
-        discovered_hosts = [(None, host.get('name'), host, None) for host in self._api_client.list_hosts(cluster_name)]
-        self._log.trace("Cloudera full hosts raw response:\n%s", discovered_hosts)
+    def _collect_hosts(self, cluster_name, config):
```

Comment:
```suggestion
    def _collect_hosts(self, cluster_name, config={}):
```
Could we set `config` by default to `{}`? Are there other possible types `config` could be? This could also get rid of `if config else None`.

Reply:
We can't control if the config value coming to us from Discovery is None or not, so the check